frontend/pages/group/data/recipes.vue

```diff
@@ -314,8 +342,6 @@ export default defineComponent({
 
       const { response, data } = await api.bulk.bulkDelete({ recipes });
```

Comment:
```suggestion
      await api.bulk.bulkDelete({ recipes });
```

[sand] unused variables
![image](https://github.com/user-attachments/assets/6c06ce53-cd0d-47a6-b09a-2b2c0cb14279)